.github/workflows/publish.yml

```diff
@@ -8,26 +8,39 @@ on:
 jobs:
   publish:
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+
+      - name: Setup node
+        uses: actions/setup-node@v2
         with:
-          version: 12
-      - run: npm ci
-      - run: npm test
-      - run: npm run build
-      - run: npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          node-version: 15.x
```

Comment:
should we stick with even node version? given that odd node version will be EOL once next major version is release. https://nodejs.org/en/about/releases/

Reply:
v14/v16 seemed like a good candidate given both have LTS till end of 2023

Reply:
I agree, but since I followed (almost exactly) what we do in react-components, let's do this change in both places is separate PRs for easier trackability.